Sources/BraintreePayPal/RecurringBillingMetadata/BTPayPalRecurringBillingDetails.swift

```diff
@@ -37,7 +39,8 @@ public struct BTPayPalRecurringBillingDetails {
         oneTimeFeeAmount: String?,
         shippingAmount: String?,
         productAmount: String?,
-        taxAmount: String?
+        taxAmount: String?,
+        totalAmount: String?
```

Comment:
Is this based on the final GW PR for this? I think the `productAmount` field also changed and there is a `productPrice` filed now

Reply:
Good catch. The param being sent is `"product_price"` while the interface is `productAmount`. I'll update this to use `productPrice`.
```
if let productAmount {
            parameters["product_price"] = productAmount
        }
```

Reply:
The SDK is sending `product_price` in the JSON blob to the GW.

I intentionally opted to name the public SDK property `amount` instead since it is more consistent with how we name dollar amounts throughout the SDK. We don't use `price` anywhere else.

Reply:
Ahh sorry my mistake!